scripts/anonymize_task_file.py

```diff
@@ -87,8 +87,8 @@ def main():
             data_dir = os.path.join(GLib.get_user_data_dir(), "gtg")
             project_filepath = os.path.join(data_dir, "projects.xml")
             dom = parse(project_filepath)
-            xmlproject = dom.getElementsByTagName("backend")[0]
-            xmlfile = str(xmlproject.getAttribute("path"))
+            xmlproject = dom.findall("//backend")[0]
```

Comment:
Would we want to skip stdlib XML in this as well and replace with LXML? If this is intended to be LXML, I didn't see the modification of the import of `xml.dom.minidom.parse`.

If I missed that and it's an LXML `_Element` or `_ElementTree`, I think you'd want to do one of these instead:

* `dom.xpath('//backend')[0]`
* `dom.find('backend')` (might only work if it's an `_Element`, not an `_ElementTree`)
* `dom.findall('backend')[0]` (might only work if it's an `_Element`, not an `_ElementTree`)

Reply:
This is one sneaky bug you found :)